deploy/roles/shared/handlers/main.yml

```diff
@@ -31,13 +31,17 @@
 - name: restart shiny-server
   command: "restart shiny-server"
 
-- name: restart postgres
-  become: yes
-  service:
-    name: "{{ pp_servicename }}"
-    state: restarted
+#- name: restart postgres
+#  become: yes
+#  service:
+#    name: "{{ pp_servicename }}"
+#    state: restarted
```

Comment:
Would this work if we were to deploy postgres without postgres-config (--tags=postgres) or that wouldn't happen?

Reply:
Handles should run even though they don't have the tag, as long as a task that does have the tag is marked "changed"